crate/cli/src/actions/shared/wrap_key.rs

```diff
@@ -75,12 +75,19 @@ impl WrapKeyAction {
             let key_bytes = general_purpose::STANDARD
                 .decode(b64)
                 .with_context(|| "failed decoding the wrap key")?;
-            create_symmetric_key(&key_bytes, CryptographicAlgorithm::AES)
+            create_symmetric_key(&key_bytes, CryptographicAlgorithm::AES)?
         } else if let Some(password) = &self.wrap_password {
             let key_bytes =
-                derive_key_from_password::<SYMMETRIC_WRAPPING_KEY_SIZE>(password.as_bytes())?
-                    .to_vec();
-            create_symmetric_key(&key_bytes, CryptographicAlgorithm::AES)
+                derive_key_from_password::<SYMMETRIC_WRAPPING_KEY_SIZE>(password.as_bytes())?;
+
+            let symmetric_key_object =
+                create_symmetric_key(&key_bytes, CryptographicAlgorithm::AES)?;
+
+            println!(
```

Comment:
is this println! still needed?

Reply:
this one is required (since this is the only time that the key will be printed)

Reply:
adding a comment